libraries/Microsoft.Bot.Connector/Authentication/BotFrameworkAuthentication.cs

```diff
@@ -79,6 +82,38 @@ public virtual Task<ClaimsIdentity> AuthenticateChannelRequestAsync(string authH
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        /// Creates a <see cref="StreamingConnection"/> that uses web sockets.
+        /// </summary>
+        /// <param name="httpContext"><see cref="HttpContext"/> instance on which to accept the web socket.</param>
+        /// <param name="logger">Logger implementation for tracing and debugging information.</param>
+        /// <returns><see cref="StreamingConnection"/> that uses web socket.</returns>
+        public virtual Task<StreamingConnection> CreateWebSocketConnectionAsync(HttpContext httpContext, ILogger logger)
+        {
+            if (httpContext == null)
+            {
+                throw new ArgumentNullException(nameof(httpContext));
+            }
+
+            return Task.FromResult<StreamingConnection>(new WebSocketStreamingConnection(httpContext, logger));
+        }
+
+        /// <summary>
+        /// Creates a <see cref="StreamingConnection"/> that uses named pipes.
+        /// </summary>
+        /// <param name="pipeName">The name of the named pipe.</param>
+        /// <param name="logger">Logger implementation for tracing and debugging information.</param>
+        /// <returns>A <see cref="StreamingConnection"/> that uses named pipes.</returns>
+        public virtual StreamingConnection CreateNamedPipeConnection(string pipeName, ILogger logger)
```

Comment:
We should not need this method, the pipe path should be unchanged for the time being.

Reply:
Reverted this method.